roles/container-engine/docker/templates/docker.service.j2

```diff
@@ -1,19 +1,11 @@
 [Unit]
 Description=Docker Application Container Engine
 Documentation=http://docs.docker.com
-{% if ansible_os_family == "RedHat" %}
-After=network.target {{ ' docker-storage-setup.service' if docker_container_storage_setup else '' }} containerd.service
+After=network.target docker.socket containerd.service lvm2-monitor.service SuSEfirewall2.service
```

Comment:
HI @VannTen 

The `SuSEfirewall2` is only needed when `ansible_os_family == "Suse" `

Reply:
This is intentional. systemd tolerates non-existent unit in After= without problem.
*not* doing a conditional After list allow a simpler template on our hand.

systemd has generally the principle of "progressive enhancement", aka, it supports what's available on the system it's running on, and ignore the rest ; so I usually find it more appropriate to have as little variation in systemd unit as possible between environments, as long as it's supported.